packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -0,0 +1,1352 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/Chart.yaml packages/rancher-monitoring/charts/Chart.yaml
+--- packages/rancher-monitoring/charts-original/Chart.yaml
++++ packages/rancher-monitoring/charts/Chart.yaml
+@@ -1,5 +1,9 @@
+ apiVersion: v1
+ appVersion: 0.38.1
++annotations:
++  catalog.rancher.io/certified: rancher
++  catalog.rancher.io/namespace: rancher-monitoring
```

Comment:
Let's update this to `monitoring-system`. UI has requested that all the namespaces end in `-system` to allow filtering.

Reply:
Updated in the latest commit